jupyterlab_wipp/src/wippRegister.ts

```diff
@@ -87,7 +67,7 @@ export class WippRegister extends Widget{
         this.spinner.dispose();
 
         // Print the response for debug
-        console.log(response);
+        console.debug(response);
```

Comment:
This message could use a little bit more context.

Reply:
Actually, this output is really not needed at all, it was left here since I developed the plugin